build.gradle

```diff
@@ -2,11 +2,14 @@ plugins {
     id 'java'
     id 'application'
     id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '7.1.0'
 }
 
 group 'org.example'
 version '1.0'
 mainClassName = "org.springframework.boot.loader.JarLauncher"
+sourceCompatibility = 11
```

Comment:
I think I had some issues with this in Jitpack. Does Jitpack still compile this project?

Reply:
I see...
```
* What went wrong:
Execution failed for task ':compileJava'.
> invalid source release: 11
```

Reply:
apparently you need to set it to java 11 in your jitpack.yml
lemme try that
https://github.com/jitpack/jitpack.io/issues/4022

Reply:
that seems to have fixed it!
[`6445d75` (#1)](https://github.com/freyacodes/lavalink-plugin-template/pull/1/commits/6445d75ebb357aad522174c04d4d07b0c1040245)

https://jitpack.io/com/github/TopiSenpai/lavalink-plugin-template/-6445d75ebb-1/build.log